<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WTempADM</class>
 <widget class="QWidget" name="WTempADM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>175</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>175</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>175</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WTempADM</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(128, 128, 128);</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>175</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192, 192, 192);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbTemADMT">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>4</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192, 192, 192);</string>
   </property>
   <property name="text">
    <string>TEMP ADM AL COMP DE B</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbADMT_1">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>31</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TE6821A</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbADM_1">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>31</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>65.00</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbADMU_1">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>31</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ºF</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbADMU_2">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>45</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ºF</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbADM_2">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>45</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>65.00</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbADMT_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>45</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TE6821B</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbFondo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>175</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>lbFondo</zorder>
  <zorder>frame</zorder>
  <zorder>lbTemADMT</zorder>
  <zorder>lbADMT_1</zorder>
  <zorder>lbADM_1</zorder>
  <zorder>lbADMU_1</zorder>
  <zorder>lbADMU_2</zorder>
  <zorder>lbADM_2</zorder>
  <zorder>lbADMT_2</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
